archive/r/ruby/README.md

```diff
@@ -36,3 +37,4 @@ Welcome to Sample Programs in Ruby!
 [9]: https://repl.it/languages/ruby
 [10]: https://github.com/TheRenegadeCoder/sample-programs/issues/433
 [11]: https://github.com/TheRenegadeCoder/sample-programs/issues/455
+[12]: 
```

Comment:
Was there supposed to be a link here?

Reply:
Yes. On my side it shows that I've added the link.